src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Methodologies/MethodologyController.cs

```diff
@@ -28,47 +28,47 @@ public class MethodologyController : ControllerBase
             _methodologyAmendmentService = methodologyAmendmentService;
         }
 
-        [HttpPut("publication/{publicationId}/methodology/{methodologyId}")]
+        [HttpPut("publication/{publicationId:guid}/methodology/{methodologyId}")]
         public async Task<ActionResult<Unit>> AdoptMethodology(Guid publicationId, Guid methodologyId)
         {
             return await _methodologyService
                 .AdoptMethodology(publicationId, methodologyId)
                 .HandleFailuresOrOk();
         }
 
-        [HttpPost("publication/{publicationId}/methodology")]
-        public Task<ActionResult<MethodologyVersionSummaryViewModel>> CreateMethodology(Guid publicationId)
+        [HttpPost("publication/{publicationId:guid}/methodology")]
+        public Task<ActionResult<MethodologyVersionViewModel>> CreateMethodology(Guid publicationId)
         {
             return _methodologyService
                 .CreateMethodology(publicationId)
                 .HandleFailuresOrOk();
         }
 
-        [HttpDelete("publication/{publicationId}/methodology/{methodologyId}")]
+        [HttpDelete("publication/{publicationId:guid}/methodology/{methodologyId}")]
         public async Task<ActionResult> DropMethodology(Guid publicationId, Guid methodologyId)
         {
             return await _methodologyService
                 .DropMethodology(publicationId, methodologyId)
                 .HandleFailuresOrNoContent();
         }
 
-        [HttpGet("publication/{publicationId}/adoptable-methodologies")]
-        public async Task<ActionResult<List<MethodologyVersionSummaryViewModel>>> GetAdoptableMethodologies(Guid publicationId)
+        [HttpGet("publication/{publicationId:guid}/adoptable-methodologies")]
+        public async Task<ActionResult<List<MethodologyVersionViewModel>>> GetAdoptableMethodologies(Guid publicationId)
         {
             return await _methodologyService
                 .GetAdoptableMethodologies(publicationId)
                 .HandleFailuresOrOk();
         }
 
-        [HttpGet("methodology/{methodologyVersionId}/summary")]
-        public async Task<ActionResult<MethodologyVersionSummaryViewModel>> GetMethodologySummary(Guid methodologyVersionId)
+        [HttpGet("methodology/{methodologyVersionId:guid}/summary")]
+        public async Task<ActionResult<MethodologyVersionViewModel>> GetMethodologySummary(Guid methodologyVersionId)
```

Comment:
We could:
- Remove the summary part of the URL i.e. `methodology/{methodologyVersionId:guid}` (requires associated frontend change to `methodologyService` too)
- Rename the method to `GetMethodology`